services/login/src/interfaces/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+export interface IUser {
+  email: string;
+  nickname: string;
+  thumbnailImageUrl: string;
+  profileImageUrl: string;
+}
+
+export interface IAxiosError {
+  code: number;
+  httpStatus: string;
+  message: string;
+  statusCode: number;
+}
```

Comment:
`extends`로 Axios에서 제공하는 기본 에러 타입을 extends 하면 좋을 것 같습니다
```suggestion
export interface CustomAxiosError extends AxiosError {
  code: number;
  httpStatus: string;
  message: string;
  statusCode: number;
}
```

Reply:
[4c2c339](https://github.com/depromeet/toks-web/pull/110/commits/4c2c3396955640f9f62b88a1e3858cf7edf019ac)
넵 이 부분 http 패키지에도 추가해두었습니다!! 근데 생각해보니까 제가 필요한 에러 타입은 response.data 타입인데 여기서는 그냥 AxiosError타입이라 코드가 string, number로 타입이 안맞아서 에러가 생기더라구요 훔 이거 그럼 AxiosError를 extends 하면 안되는걸까요?!